### DISCUSSION TITLE
My support cases are removed without any clue

### DISCUSSION THREAD

**Discussion by petdud** [vercel/community#6513] [Account Issue]
### Question

I created three help cases in my account in the past 2 days and they are being removed without any clue or information why.. I also don't see them in my Support Center.

(I created one, it was removed, then created another one and also removed... so no spam but not clue what's going on, probably you have some weird bug).

Those are the ticket items:
- 00220156
- 00219966
- 00220002

For all I also received email notification saying it's from hobby plan despite I upgraded to Premium, so I assume it's a bug.
Anyway would be great if you could help me to resolve this: https://github.com/orgs/vercel/discussions/6500 :)

**Comment by amyegan** [comment]
Hey @petdud. I see that you still have an open case in our ticket system, and multiple similar questions were merged into one case to avoid duplicate responses. It looks like you've had some responses on the remaining case since you originally posted this, so I'm going to close this discussion now. 

Please continue to respond via email to your Customer Success Engineer if you have any further questions 🙏
